ui/src/app/completed/controllers/TableCtrl.js

```diff
@@ -6,20 +6,50 @@
 
   function TableCtrl($scope, ToastService, TableService, UtilityService) {
     $scope.downloads = [];
-
     var setSize = function(lst) {
       lst.data.forEach(function(download) {
         download.size = UtilityService.formatBytes(download.size);
       })
       return lst;
     };
-
+    $scope.isGridOn = false;
```

Comment:
move $scope, var declarations to the top so that it is easy to read. All the declarations will be on top.

Reply:
ok @rehrumesh 😄